xla/service/gpu/cudnn_fused_conv_rewriter.cc

```diff
@@ -331,8 +333,8 @@ StatusOr<bool> FuseConvAlpha(HloComputation* comp) {
 
     TF_ASSIGN_OR_RETURN(Literal alpha_f64, alpha->literal().Convert(F64));
     config.set_conv_result_scale(alpha_f64.GetFirstElement<double>());
-
-    TF_RETURN_IF_ERROR(conv->set_backend_config(config));
+    *gpu_config.mutable_cudnn_conv_backend_config() = config;
```

Comment:
ditto and same applies to all other applicable locations (not commenting on every single one since there are a lot)